src/BetterPhpDocParser/PhpDoc/DoctrineAnnotationTagValueNode.php

```diff
@@ -18,11 +19,16 @@ public function __construct(
         public IdentifierTypeNode $identifierTypeNode,
         ?string $originalContent = null,
         array $values = [],
-        ?string $silentKey = null
+        ?string $silentKey = null,
+        ?string $comment = null
     ) {
         $this->hasChanged = true;
 
         parent::__construct($values, $originalContent, $silentKey);
+
+        if ($comment) {
```

Comment:
When creating the tag value node, if the tag had a comment, we save it as an attribute in the node to be used later

Reply:
this should be strict whether "0" or empty string "" or null, I guess you can do:

```php
if (! in_array($comment, ["", null], true))
```

since "0" comment is valid